### DISCUSSION TITLE
Annotating  web pages in Zotero

### DISCUSSION THREAD

**Discussion by Bekichen** [BryanJenksCommunity/FAQ#183] [Q&A] [answered]
Hi Brian - Do you have a process for automatically saving snapshots to PDFs in zotero so that they can be read, annotated and imported into obsidian? I have not found a satisfactory solution. Cheers

**Comment by Bekichen** [comment]
I have worked out how to annotate the snapshots - but these annotations do not get imported into obsidian.

**Comment by tallguyjenks** [comment]
i do not take snapshots into zotero so i have no workflow in place but if you got a PDF into zotero, it's odd that the annotations are not being extracted

**Comment by Bekichen** [comment, marked as answer]
Thanks Brian. The snapshot is html and the zotero import into obsidian only looks for annotations and comments on pdfs from what I can tell? I will resort to the few additional steps of saving and attaching the snapshot to the zotero parent item as a pdf until someone develops an extension to complete this step automatically 😊Sent from my iPhoneOn 6 Dec 2023, at 10:20 am, Bryan Jenks ***@***.***> wrote:﻿
i do not take snapshots into zotero so i have no workflow in place but if you got a PDF into zotero, it's odd that the annotations are not being extracted
